.gitignore

```diff
@@ -176,6 +176,7 @@ dist
 wrangler-dist/
 miniflare-dist
 packages/wrangler/wrangler.toml
+packages/wrangler/message.capnp.compiled
```

Comment:
odd. this shouldn't even be necessary, because we run the publish tests in a temp dir.

Reply:
I might be doing something wrong, but I didn't see any other tests explicitly changing the directory?

Reply:
`runInTemp` has everything in the test run in a directory that is removed. If you need examples of a test that changes directories during a test closure, anything using `setWorkingDirectory("some-name")` is doing a `process.chdir()`

Reply:
I tried this out locally, and it doesn't output directly into this directory, so isn't needed. I'm going to remove it myself.